keyboards/keyprez/corgi/readme.md

```diff
@@ -0,0 +1,25 @@
+# corgi
+
+![corgi](https://i.imgur.com/XctcpVr.jpeg)
+
+*Corgi is a semi modular 4 x 12 ortho keyboard. The unique thing with Corgi is that you can attach a macropad name "Flea" to use as a num/macro-pad.*
+
+* Keyboard Maintainer: [Christian Sandven](https://github.com/csandven)
+* Hardware Supported: *Pro Micro or Elite C is required*
+* Hardware Availability: *TBA*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keyprez/corgi:default
+
+Flashing example for this keyboard:
+
+    make keyprez/corgi:default:flash
+    
+Flashing for Pro micro:
+    
+    make keyprez/corgi:default:avrdude
```

Comment:
This isn't needed.  `:flash` goes off the bootloader that is set, so changing the bootloader to `caterina` will use the correct flashing method.

Also, you want to have the bootloader set correctly, because different bootloaders handle the jump to bootloader mode different.  So if it's set wrong, it will just reset to the firmware, when using the `RESET` keycode, or bootmagic. 
```suggestion
```